uiowa/uiowa
# Grad College and usability updates

# Updates
## Grad College
* Pulled in functionality to allow defining a main list of programs for the Grad college and to filter it for allowed values lists from #2527.
* Renamed field to better align with the allowed values.
* Update Grad programs and certificates view displays.
* Updated `drupal/fragments` to 1.0.0-beta3, which is needed for #2474. Seemed like a good time to do this, since this modules usage is limited to the GC split.
* Switched scholarship content type to use a Layout Builder template.

## Platform
* Added a menu route to 'Add content' that can be accessed from the admin toolbar without going to the Content overview page. Only accessible when the menu is showing on the side.
  ![Screen Shot 2021-01-14 at 10 25 52 PM](https://user-images.githubusercontent.com/709009/104681256-84167400-56b7-11eb-87a9-2d8cf7645d97.png)
* Added `field--title--node--scholarship--default.html.twig` template.

# How to test

* `blt ds --site=grad.uiowa.edu`
* `drush @grad.local uli`
* Note no problems with import.
* Check that the menu shows as pictured above.

## Check Admissions scholarship titles are displaying correctly
* `blt ds --site=admissions.uiowa.edu`
* `drush @grad.local uli scholarship/advantage-iowa-award` and check that heading looks like https://admissions.prod.drupal.uiowa.edu/scholarship/advantage-iowa-award
